ssl/d1_lib.c

```diff
@@ -331,7 +356,7 @@ void dtls1_stop_timer(SSL *s)
     /* Reset everything */
     memset(&s->d1->timeout, 0, sizeof(s->d1->timeout));
     memset(&s->d1->next_timeout, 0, sizeof(s->d1->next_timeout));
-    s->d1->timeout_duration = 1;
+    s->d1->timeout_duration_ms = 1000;
```

Comment:
> Before it was 1 (second)

Yes, but now it should be 0 to indicate "not set". Actually, I don't think anything reads this anyway.